constraint/config/crd_patch.yaml

```diff
@@ -3,13 +3,4 @@ kind: CustomResourceDefinition
 metadata:
   name: constrainttemplates.templates.gatekeeper.sh
 spec:
-  subresources:
-    status: {}
-  versions:
-    - name: v1beta1
-      served: true
-      storage: true
-    - name: v1alpha1
-      served: true
-      storage: false
-
+  version: v1beta1
```

Comment:
Would love to get rid of this completely, but I'm not sure how.

@maxsmythe you know the kubebuilder annotations pretty well... do you know how to change it?

That said, all the v1beta1 CRD examples in [this kubernetes documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/) don't even include the `version:` field, so maybe we can just remove it.

Reply:
Yeah, the version field just needs to equal the first item in the `versions` list, if present.

From the looks of it, this can be removed.

Reply:
It looks like leaving it in is actually breaking the unit tests

Reply:
Removed it using kustomize